### PR TITLE
add k8s podcast

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: src/posts/2021-01-12-k8s-pocast/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+date: '2021-01-12T17:00:00.000Z'
+title: 'Cilium with Thomas Graf on Google Kubernetes Podcast'
+ogImageUrl: 'https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4_400x400.jpg'
```
Comment (liquidat) [inline src/posts/2021-01-12-k8s-pocast/index.md:4]: Replace with this image:
https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4.jpg

**Review by liquidat — CHANGES_REQUESTED** [review]
Please update image to the larger version which has no compressing artifacts.

File: src/posts/2021-01-12-k8s-pocast/index.md (hunk 1 above)
Comment (xmulligan) [inline src/posts/2021-01-12-k8s-pocast/index.md:4]:
```suggestion
ogImageUrl: 'https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4.jpg'
```

File: src/posts/2021-01-12-k8s-pocast/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+date: '2021-01-12T17:00:00.000Z'
+title: 'Cilium with Thomas Graf on Google Kubernetes Podcast'
+ogImageUrl: 'https://pbs.twimg.com/profile_images/1095059886730137600/icMSa8g4.jpg'
+ogSummary: 'Thomas joins the Kubernetes Podcast from Google to talk about Cilium'
+categories:
+  - Technology
```
Comment (xmulligan) [inline src/posts/2021-01-12-k8s-pocast/index.md]:
```suggestion
  - Community
```

**Review by liquidat — APPROVED** [review]

**Pull request merged**